src/operator/time_series/rolling_aggregate.rs

```diff
@@ -133,6 +200,49 @@ impl<B> Stream<Circuit<()>, B> {
             output
         })
     }
+
+    /// A version of [`Self::partitioned_rolling_aggregate`] optimized for
+    /// linear aggregation functions.
+    ///
+    /// This method only works for linear aggregation functions `f`, i.e.,
```

Comment:
but linear in the group of result values, not in the ZSet group

Reply:
I think it's linear in both.